Classes/label.rb

```diff
@@ -0,0 +1,16 @@
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items

```

Comment:
With` attr_accessor` we can read and write property value. I think all properties of the Label class should be set up in the constructor method. I suggest you change `attr_accessor :title, :color`  to `attr_reader :title, :color` .

Reply:
@developertariq Thank you for the suggestions, but all the properties are already defined in the constructor i.e., initialize method in Ruby. attr_accessor is only used to read as well as to write these properties as you already said.